background.js

```diff
@@ -49,47 +91,45 @@ const waitAndClick = async (querySelector) => {
   if (await waitForElement(querySelector)) await clickElement(querySelector);
 };
 
-browser.pageAction.onClicked.addListener(async () => {
-  await browser.tabs.executeScript({
+browser.pageAction.onClicked.addListener(async (tab) => {
```

Comment:
What would happen if the user clicks the extension twice? Is this code free of race conditions?

Reply:
Now there is additional state. If user clicked `pageAction`, second call wouldn't be executed